drools-parser/src/test/java/org/drools/parser/MiscDRLParserTest.java

```diff
@@ -819,4 +854,2433 @@ void parse_multipleRestrictionsConstraint() throws Exception {
         fld = (ExprConstraintDescr) and.getDescrs().get( 1 );
         assertThat(fld.getExpression()).isEqualToIgnoringWhitespace( "age < 3");
     }
+
+//-------------------------------------------------------------------------
+// DROOLS-7271 : ported from RuleParserTest
+// Failing tests are annotated with @Disabled. We can fix issues one by one
+//-------------------------------------------------------------------------
+
+    @Disabled("Priority : Mid | implement Descr lineNumber")
+    @Test
+    public void parse_LineNumberInAST() throws Exception {
+        // also see testSimpleExpander to see how this works with an expander
+        // (should be the same).
+
+        final RuleDescr rule = parseAndGetFirstRuleDescrFromFile(
+                                                          "simple_rule.drl" );
+
+        assertThat(rule).isNotNull();
+
+        assertThat(rule.getName()).isEqualTo("simple_rule");
+
+        assertThat(rule.getConsequenceLine()).isEqualTo(22);
+        assertThat(rule.getConsequencePattern()).isEqualTo(2);
+
+        final AndDescr lhs = rule.getLhs();
+
+        assertThat(lhs).isNotNull();
+
+        assertThat(lhs.getDescrs().size()).isEqualTo(3);
+
+        // Check first pattern
+        final PatternDescr first = (PatternDescr) lhs.getDescrs().get( 0 );
+        assertThat(first.getIdentifier()).isEqualTo("foo3");
+        assertThat(first.getObjectType()).isEqualTo("Bar");
+        assertThat(first.getConstraint().getDescrs().size()).isEqualTo(1);
+
+        // Check second pattern
+        final PatternDescr second = (PatternDescr) lhs.getDescrs().get( 1 );
+        assertThat(second.getIdentifier()).isEqualTo("foo4");
+        assertThat(second.getObjectType()).isEqualTo("Bar");
+
+        final PatternDescr third = (PatternDescr) lhs.getDescrs().get( 2 );
+        assertThat(third.getObjectType()).isEqualTo("Baz");
+
+        assertThat(first.getLine()).isEqualTo(19);
+        assertThat(second.getLine()).isEqualTo(20);
+        assertThat(third.getLine()).isEqualTo(21);
+    }
+
+    @Disabled("Priority : High | Failed to parse comments in RHS")
+    @Test
+    public void parse_LineNumberIncludingCommentsInRHS() throws Exception {
```

Comment:
For example, I confirmed that this test fails so I filed https://issues.redhat.com/browse/DROOLS-7287 as a sub-task to fix this issue. When I will raise a PR with a fix, I'll also remove this `@Disabled`.